### PR TITLE
Update Install_with_flatpak.md

### PR DESCRIPTION
**Pull request by thiblahute**

### CODE REVIEW

File: docs/Install_with_flatpak.md [hunk 1]
```diff
@@ -24,7 +24,11 @@ only one time.
 To install the latest stable Pitivi release, run as a normal user (no
 root nor sudo):
 
-` $ flatpak install --user `[`http://flatpak.pitivi.org/pitivi.flatpakref`](http://flatpak.pitivi.org/pitivi.flatpakref)
+  `$ flatpak install --user `[`https://flathub.org/repo/appstream/org.pitivi.Pitivi.flatpakref`](https://flathub.org/repo/appstream/org.pitivi.Pitivi.flatpakref)
+
+We also provide latest development snapshot toward what is going to become `Pitivi 1.0`, you can install with:
```
Comment (aleb) [inline docs/Install_with_flatpak.md]: This would go in the "Installing Pitivi master" section. Maybe move it here..

Comment (thiblahute) [reply docs/Install_with_flatpak.md]: Not sure, it is not master but a "stablish" version xD

Comment (thiblahute) [reply docs/Install_with_flatpak.md]: What should we do about that?

**Comment by thiblahute** [comment]
https://phabricator.freedesktop.org/T7894

**Pull request closed without merging**